lib/service/Service.ts

```diff
@@ -48,13 +51,54 @@ class Service extends EventEmitter {
     this.shutdown = components.shutdown;
 
     this.orderBook = components.orderBook;
-    this.lndClient = components.lndClient;
+    this.lndBtcClient = components.lndBtcClient;
+    this.lndLtcClient = components.lndLtcClient;
     this.raidenClient = components.raidenClient;
     this.pool = components.pool;
     this.config = components.config;
 
     this.logger = Logger.rpc;
   }
+  private getLndInfo = async (lndClient: LndClient)  => {
+
```

Comment:
Can you drop this blank line, and add one above `private getLndInfo` to space it from the close bracket above it? Just for consistent spacing.

Reply:
done.